test/hts-precompile/utils.js

```diff
@@ -623,7 +627,7 @@ class Utils {
     if (setSupply) tx.setSupplyKey(keyList);
     if (setWipe) tx.setWipeKey(keyList);
 
-    await (
+    return await (
```

Comment:
Was wondering why these changes are here in this PR? Is it related to the local-node being bumped to 2.25.2?

Reply:
Those changes were left by mistake